CPT331.WebAPI/Api/TestController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+
+namespace CPT331.WebAPI.Api
+{
+    public class TestController : ApiController
```

Comment:
@s3364859  I like to keep API Controllers and MVC (webpage view) Controllers separate.

It makes it easier to maintain when a project implements both an API and a website.

This is simply a convention we use at work, and I'm happy to change it if needed.